Car_Collision_Visulaization/src/car_collision.h

```diff
@@ -9,8 +12,11 @@ class Car
     public:
     double x_, y_ ;         
     double velocity_,angle_;
-
+    Car();
     explicit Car(double, double, double, double);
+    void move();
```

Comment:
please define variables and functions separately
public: 
  double x_;
  double y_;
  double velocity_;
  
public:
  explicit Car (double x, double y, double velocity, double angle);
  void move();
  bool isColliding(const Car& target_vehicle);

Reply:
Car should have length and width as well